packages/bvaughn-architecture-demo/components/sources/AutoComplete/AutoCompleteList.tsx

```diff
@@ -89,18 +91,17 @@ function AutoCompleteListInner({
   }, [expression]);
 
   let properties: Property[] | null = null;
-  let scopes: Scope[] | null = pause.data.scopes || null;
+  const frameId = getFramesSuspense(replayClient, pauseId)?.[0]?.frameId ?? null;
+  const scopes = frameId ? getScopesSuspense(replayClient, pauseId, frameId).generatedScopes : null;
```

Comment:
Should this be pulling frame from the new `SelectedFrameContext` rather than just snagging the top one?